app/assets/stylesheets/govuk-component/_govspeak-html-publication-print.scss

```diff
@@ -1,5 +1,3 @@
-.govuk-govspeak-html-publication {
-  .govuk-sticky-element {
-    display: none;
-  }
+.govuk-sticky-element {
```

Comment:
I think the contents link in government-frontend needs a `dont-print` class added to. But I agree – it's separate to this PR.